documentation/modules/managing/proc-drain-cleaner-deploying.adoc

```diff
@@ -93,43 +96,39 @@ Add the same configuration for ZooKeeper if you want to use Strimzi Drain Cleane
 . Update the `Kafka` resource:
 +
 [source,shell,subs=+quotes]
-kubectl apply -f _<kafka-configuration-file>_
+kubectl apply -f <kafka_configuration_file>
 
 . Deploy the Strimzi Drain Cleaner.
 +
-ifdef::Section[]
 --
-You can use link:https://cert-manager.io/docs/[`cert-manager`^] in the deployment process.
-
-* If you are using `cert-manager` with Kubernetes, apply the resources in the `/install/drain-cleaner/certmanager` directory.
+ifdef::Section[]
+* If you are using `cert-manager` with Kubernetes, apply the resources in the `/install/certmanager` directory.
 +
 [source,shell,subs="attributes+"]
 ----
-kubectl apply -f ./install/drain-cleaner/certmanager
+kubectl apply -f ./install/certmanager
```

Comment:
In the Strimzi installation files, all Drain Cleaner installation files are in `install/drain-cleaner/...`. So this should keep the original path we had here.